src/constants/index.ts

```diff
@@ -76,7 +83,16 @@ export const BTC_SWAP_TOKEN = new Token(
   18,
   "BLPT",
   "blpt",
-  "Saddle BTC Pool LP Token",
+  "Saddle BTC LP Token",
+  saddleLogo,
+)
+
+export const STABLECOIN_SWAP_TOKEN = new Token(
+  STABLECOIN_SWAP_TOKEN_CONTRACT_ADDRESSES,
+  18,
+  "SLPT",
```

Comment:
I'm curious about the correctness of these names / symbols

Reply:
For reference, on-chain USD LP token will follow the btc lp token's convention
`Saddle DAI/USDC/USDT` for the name
`saddleUSD` for symbol

Reply:
Thanks, will update!